events/static/js/app/event.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Map from './map.js'
+
+
+class Event extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = ({events: {}})
+    }
+    componentWillMount(){
+        const url = '/api/v1/events/' + this.props.params.event_id
+        axios.get(url) 
+        .then(function (response) {
+            const events_array = []
+            events_array.push(response.data)
+            this.setState({events: events_array})
+        }.bind(this))
+        .catch(function (error) {
+            console.log(error);
```

Comment:
How many users do you expect will see this error? Usually, a user expects to see errors on ui rather than console.